data-prepper-plugins/kafka-plugins/src/main/java/org/opensearch/dataprepper/plugins/kafka/configuration/KafkaSourceConfig.java

```diff
@@ -51,6 +51,26 @@ public class KafkaSourceConfig {
     @JsonProperty("acknowledgments_timeout")
     private Duration acknowledgementsTimeout = DEFAULT_ACKNOWLEDGEMENTS_TIMEOUT;
 
+    @JsonProperty("serde_format")
+    private String serdeFormat;
+
+    @JsonProperty("client_dns_lookup")
+    private String clientDnsLookup;
```

Comment:
If we want to support this, we should make it enum that only accepts the values `default` or `use_all_dns_ips` or `resolve_canonical_bootstrap_servers_only`

Reply:
We made it as an ENUM.